starter_code/views/beers.hbs

```diff
@@ -0,0 +1,34 @@
+
+   {{!-- <ul>
+       {{#each beers}}
+        <li>
+         <h2>{{beers.name}}</h2> 
+         <h3>{{beers.tagline}}</h3>
+         <p>{{beers.description}}</p>
+        <img src="{{this.image_url}}" >
+        </li>
+       {{/each}}
+    </ul> --}}
+<div class="container"></div>
+<div class="row">
+ {{#each beers}}
```

Comment:
Here you could have used the beer partial that you created to make the code shorter :)